package.json

```diff
@@ -10,7 +10,8 @@
       "default": "./dist/main.js"
     },
     "./dist/": "./dist/",
-    "./icons": "./dist/icons/index.js"
+    "./icons": "./dist/icons/index.js",
+    "./storybook": "./dist/storybook/storybook-esm.js"
```

Comment:
let's change it to "/storybookUtils/components" because we will probably will export another services and the storybook addon  (if we will change after merge it will be breaking change)

Reply:
Not only components there, also functions and decorators. "/storybookUtils" then?
But maybe we want to make it all on one entry somehow? All of this will be needed all together, I guess

Reply:
we decide to replace it with storybookComponents